cli/src/action/circuit/builder.rs

```diff
@@ -315,6 +321,10 @@ impl CreateCircuitMessageBuilder {
             create_circuit_builder = create_circuit_builder.with_circuit_version(circuit_version);
         }
 
+        if let Some(circuit_status) = self.circuit_status {
```

Comment:
Isn’t this for creating proposals?  Why does the status need to be part of the proposal?

Reply:
I explained the reasoning for the status being in a proposal in a response to another comment, if you have further questions, let me know.

Reply:
It's checking here first if this value is set to allow for 0.4 compatibility, when creating a proto version of this struct.